python/mxnet/_numpy_op_doc.py

```diff
@@ -92,37 +111,50 @@ def _np_zeros_like(a):
 
 
 def _np_repeat(a, repeats, axis=None):
-    """Repeat elements of an array.
+    """
+    repeat(a, repeats, axis=None)  
+
+    Repeat elements of an array.
 
     Parameters
     ----------
-    a : ndarray
+    a : array_like
         Input array.
     repeats : int or array of ints
         The number of repetitions for each element.  `repeats` is broadcasted
         to fit the shape of the given axis.
     axis : int, optional
         The axis along which to repeat values.  By default, use the
         flattened input array, and return a flat output array.
-
+    
     Returns
     -------
     repeated_array : ndarray
         Output array which has the same shape as `a`, except along
         the given axis.
+
+    See Also
+    --------
+    tile : Tile an array.
+
+    Examples
+    --------
+    >>> x = np.array([[1,2],[3,4]])
+    >>> np.repeat(x, 2)
+    array([1, 1, 2, 2, 3, 3, 4, 4])
```

Comment:
My program output is of float32 type:
``array([1., 1., 2., 2., 3., 3., 4., 4.])``

Reply:
Changed. Thx.